packages/create-flex-plugin/templates/ts/src/DemoPlugin.tsx

```diff
@@ -15,15 +16,31 @@ export default class {{pluginClassName}} extends FlexPlugin {
    * This code is run when your plugin is being started
    * Use this to modify any UI components or attach to the actions framework
    *
-   * @param flex { typeof import('@twilio/flex-ui') }
-   * @param manager { import('@twilio/flex-ui').Manager }
+   * @param flex { typeof Flex }
+   * @param manager { Flex.Manager }
```

Comment:
Same as https://github.com/twilio/flex-plugin-builder/pull/115/files#diff-81a72a417692ff2031d3b5d60e6659f5R20

Reply:
Even here? This _is_ a Tyepscript project after all!

Reply:
Oh right! then you don't need jsdoc at all